go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/smartcontractkit/chainlink-data-streams
 
-go 1.21
+go 1.22
+
+toolchain go1.22.6
```

Comment:
Can we avoid bumping versions in this PR?

Reply:
I kept the go and toolchain versions the same as [`chainlink-common`](https://github.com/smartcontractkit/chainlink-common/blob/main/go.mod#L5).